docs/Features/UI.md

```diff
@@ -54,18 +65,18 @@ Further minor graphical tweaks were made, such as properly centering AP/SP displ
 
 ![Player status bar showcase](img/ui/status_console.png)
 
-## Player Portraits
+## Player portraits
```

Comment:
For headers I would prefer all nouns to start with uppercase, tbh. Looks more elegant